packages/env-options/src/env-options.js

```diff
@@ -51,12 +51,11 @@ export const makeEnvironmentCaptor = (aGlobal, dropNames = false) => {
    * Gets an environment option by name and returns the option value or the
    * given default.
    *
-   * @template {string} [T=string]
    * @param {string} optionName
-   * @param {T} defaultSetting
-   * @param {T[]} [optOtherValues]
+   * @param {string} defaultSetting
+   * @param {string[]} [optOtherValues]
    * If provided, the option value must be included or match `defaultSetting`.
-   * @returns {T}
+   * @returns {string}
```

Comment:
I don’t recall why this was a template type before, but there was probably a reason, @erights?